eldritch/test_events.py

```diff
@@ -111,6 +111,36 @@ def testMoveOneSpace(self):
     self.assertEqual(self.char.place.name, "Easttown")
     self.assertEqual(self.char.movement_points, 3)
 
+  def testMoveOneSpaceToMonster(self):
+    while self.state.turn_phase != "movement":
+      self.state.next_turn()
+    maniac = next(monster for monster in self.state.monsters if monster.name == "Maniac")
+    maniac.place = self.state.places["Easttown"]
+    movement = Movement(self.char, [self.state.places["Easttown"]])
```

Comment:
You'll have to rewrite this to use CityMovement - sorry.

Reply:
I like CityMovement better now that we have advance_to_phase!